HousingFinanceInterimApi.Tests/V1/Infrastructure/DatabaseContext/LoadCashFileTests.cs

```diff
@@ -21,6 +21,7 @@ public class LoadCashFileTests : IClassFixture<BaseContextTest>
 
         public LoadCashFileTests(BaseContextTest baseContextTest)
         {
+            System.Threading.Thread.CurrentThread.CurrentCulture = new System.Globalization.CultureInfo("en-GB");
```

Comment:
What is that?

Reply:
It makes sure DateTime.ParseDate checks ddMMYYYY (UK format) instead of the default mmDDyyyy (US format)

Reply:
Why is it against System.Threading though?